docker/env_file_app_template

```diff
@@ -33,6 +33,7 @@ ELASTICSEARCH_NO_OF_SHARDS=1
 ELASTICSEARCH_NO_OF_REPLICAS=0
 
 # analyzers secrets (optional)
+BITCOINABUSE_KEY=
```

Comment:
also add `BITCOINABUSE_KEY=test` to the `env_file_app_ci` file.

Reply:
i missed this before

Reply:
Done!